pub/css/main.css

```diff
@@ -198,39 +402,319 @@ textarea {
     width: auto;
 }
 
+/*
+ * Hide visually and from screenreaders, but maintain layout
+ */
+
 .invisible {
     visibility: hidden;
 }
 
+/*
+ * Clearfix: contain floats
+ *
+ * For modern browsers
+ * 1. The space content is one way to avoid an Opera bug when the
+ *    `contenteditable` attribute is included anywhere else in the document.
+ *    Otherwise it causes space to appear at the top and bottom of elements
+ *    that receive the `clearfix` class.
+ * 2. The use of `table` rather than `block` is only necessary if using
+ *    `:before` to contain the top-margins of child elements.
+ */
+
 .clearfix:before,
 .clearfix:after {
-    content: " ";
-    display: table;
+    content: " "; /* 1 */
+    display: table; /* 2 */
 }
 
 .clearfix:after {
     clear: both;
 }
 
+/*
+ * For IE 6/7 only
+ * Include this rule to trigger hasLayout and contain floats.
+ */
+
 .clearfix {
     *zoom: 1;
 }
 
 /* ==========================================================================
-   Print styles
+   EXAMPLE Media Queries for Responsive Design.
+   These examples override the primary ('mobile first') styles.
+   Modify as content requires.
+   ========================================================================== */
+@media only screen and (max-width: 678px) {
+    .hidden-mobile {
+        display: none;
+    }
+}
+
+@media only screen and (min-width: 678px) {    
```

Comment:
Wrapping the _entire_ stylesheet in a media query is a bit of overkill. With a _whatever_-first approach in mind, the default styles in your CSS file should be geared towards that _whatever_ -- mobile, tablet or desktop -- and unburdened by a resolution selector. The MQs come into play by _overriding those defaults_ via the cascading rules when the viewport or device size matches the query. I hope the discussion today shed some light on that part of RWD for you... The way MQs work with the cascade is fundamental to building efficient responsive layouts.